src/content/components/search/usage.mdx

```diff
@@ -29,6 +26,8 @@ _Search_ enables users to specify a word or a phrase to find particular relevant
 
 Set users' context for the search with helpful placeholder text within the search field (search documents), that pertains to the page or section the search box is in. Search in the global header should just say search.
 
+Accessibility note: If the search component's label and placeholder text differ, VoiceOver on Mac OSX will read both.
```

Comment:
```suggestion
Accessibility note: If the search component's label and placeholder text differ, voiceover on macOS will read both.
```

Reply:
I accepted the change for "macOS" but reverted "VoiceOver" as that is the program's official name